server/src/internalClusterTest/java/org/opensearch/discovery/DiscoveryDisruptionIT.java

```diff
@@ -81,7 +81,7 @@ public void testClusterJoinDespiteOfPublishingIssues() throws Exception {
         );
         nonClusterManagerTransportService.addFailToSendNoConnectRule(clusterManagerTranspotService);
 
-        assertNoMaster(nonClusterManagerNode);
+        assertNoClusterManager(nonClusterManagerNode);
```

Comment:
`stopRandomNonMasterNode`, `getMasterName`

Reply:
Same as above, `stopRandomNonMasterNode` and `getMasterName` are both public methods in `test/framework` module. 😁